quickembeddedshellwindow/waylandinputregion.cpp

```diff
@@ -34,11 +34,19 @@ void WaylandInputRegion::itemChange(ItemChange c, const ItemChangeData& d) {
     QQuickItem::itemChange(c,d);
 }
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
 void WaylandInputRegion::geometryChanged(const QRectF &newGeometry,
                                          const QRectF &oldGeometry) {
-  QQuickItem::geometryChanged(newGeometry, oldGeometry);
-  updateRegion();
+    QQuickItem::geometryChanged(newGeometry, oldGeometry);
+    updateRegion();
+}
+else
```

Comment:
```suggestion
#else
```
I wonder why this hasn't caused an compilation error.

Reply:
You are right, I don't know. But on my machine it works :)